modules/osdk-suggested-namespace-node-selector.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-suggested-namespace-default-node_{context}"]
+= Setting a suggested namespace with default node selector
+
+Some Operators expect to run only on primary nodes, which can be done by setting a `nodeSelector` in the Pod spec by the Operator itself.
```

Comment:
> Some Operators expect to run only on _**primary**_ nodes,

This should be "master" nodes, which is one of the node types of an OpenShift/Kubernetes cluster.  Sometimes, "master" nodes and "control plane" nodes are interchangeable, see in OCP doc:
https://docs.openshift.com/container-platform/4.12/nodes/nodes/nodes-nodes-managing.html#nodes-nodes-working-master-schedulable_nodes-nodes-jobs